services/device-registry/src/test/java/org/eclipse/hono/deviceregistry/FileBasedCredentialsServiceTest.java

```diff
@@ -454,6 +491,7 @@ private static void register(
             final String deviceId,
             final String authId,
             final String type,
+            final JsonObject clientContext,
```

Comment:
can you also please add a variant of this method without the clientContext param and remove all the ugly `null` passing invocations from the test cases again?